indico/modules/events/registration/client/js/form/fields/PictureInput.jsx

```diff
@@ -0,0 +1,85 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2024 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+import previewURL from 'indico-url:event_registration.registration_picture';
+import uploadURL from 'indico-url:event_registration.upload_registration_file';
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import {useSelector} from 'react-redux';
+
+import {FinalPictureManager} from 'indico/react/components';
+import {FinalInput, validators as v} from 'indico/react/forms';
+import {Translate} from 'indico/react/i18n';
+
+import {getUpdateMode} from '../../form_submission/selectors';
+import {getStaticData} from '../selectors';
+
+import '../../../styles/regform.module.scss';
+import './FileInput.module.scss';
+
+export default function PictureInput({htmlName, disabled, isRequired, minPictureSize}) {
+  const {eventId, regformId, registrationUuid, fileData} = useSelector(getStaticData);
+  const isUpdateMode = useSelector(getUpdateMode);
+  const initialPictureDetails = isUpdateMode ? fileData[htmlName] || null : null;
```

Comment:
Would it be okay instead of checking the isUpdateMode to change this to:

`const initialPictureDetails = fileDate ? fileData[htmlName] || null : null;`

Reply:
would be OK i guess...

Reply:
okay then I've changed it